packages/ui/src/app/pages/WorkingGroups/WorkingGroups.tsx

```diff
@@ -11,10 +12,26 @@ import { WorkingGroupsTabs } from './components/WorkingGroupsTabs'
 export const WorkingGroups = () => {
   const { groups } = useWorkingGroups()
 
+  const sortedGroups = useMemo(
+    () =>
+      !groups
```

Comment:
My bad it should still be:
```suggestion
      !groups.length
```

Reply:
I've remembered to change it in deps array, but forgot to change it here :D